### PR TITLE
feat: onboarding loading

### PR DESCRIPTION
**Pull request by sshanzel**
## Changes
- Animation before redirecting the user.
- Preview on Slack: https://dailydotdev.slack.com/archives/C06K5KAHM0X/p1709693669923399

Currently deployed at preview3.

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

MI-205 #done

### CODE REVIEW

File: packages/shared/src/components/auth/PreparingYourFeed.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import LogoText from '../../svg/LogoText';
+import classed from '../../lib/classed';
+
+interface PreparingYourFeedProps {
+  isAnimating: boolean;
+}
+
+export const OnboardingContainer = classed(
+  'div',
+  'flex flex-col overflow-x-hidden items-center min-h-[100vh] w-full h-full max-h-[100vh] flex-1 z-max',
```
Comment (rebelchris) [inline packages/shared/src/components/auth/PreparingYourFeed.tsx:13]: Just wondering did we not fix these 100vh to be able to use tailwinds ones?

File: packages/shared/src/components/auth/PreparingYourFeed.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+export function PreparingYourFeed({
+  isAnimating,
+}: PreparingYourFeedProps): ReactElement {
+  return (
+    <OnboardingContainer className="justify-center">
+      <span className="mb-10 flex flex-row gap-3">
+        <LogoIcon className="w-16" />
+        <LogoText className="w-32" />
+      </span>
+      <span className="flex h-1.5 w-full max-w-[19.125rem] flex-row items-center rounded-12 bg-theme-divider-tertiary px-0.5">
```
Comment (rebelchris) [inline packages/shared/src/components/auth/PreparingYourFeed.tsx:25]: Do we want to maybe start using the new named colors here already?

**Review by rebelchris — COMMENTED** [review]
Main question, should this not be an experiment?

**Review by nensidosari — CHANGES_REQUESTED** [review]
Looks great, please take in consideration Chris comments 🙏 

Last thing from me is, please move all changes behind `onboarding_fake_loading` feature flag - boolean value.

File: packages/shared/src/components/auth/PreparingYourFeed.tsx (hunk 2 above)
Comment (sshanzel) [reply packages/shared/src/components/auth/PreparingYourFeed.tsx:25]: Yes, it was just not working when I was working with this, so I applied the correct working one.

File: packages/shared/src/components/auth/PreparingYourFeed.tsx (hunk 1 above)
Comment (sshanzel) [reply packages/shared/src/components/auth/PreparingYourFeed.tsx:13]: I just copy-pasted this from the other file, but sure I will check it.

**Comment by sshanzel** [comment]
> Main question, should this not be an experiment?

Yes, I just received a reply from Sab.

File: packages/webapp/pages/onboarding.tsx [hunk 3]
```diff
@@ -394,12 +397,7 @@ export function OnboardPage(): ReactElement {
   }
 
   if (finishedOnboarding) {
-    return (
```
Comment (nensidosari) [inline packages/webapp/pages/onboarding.tsx:397]: Did we ever use this return?

File: packages/shared/src/lib/featureManagement.ts [hunk 4]
```diff
@@ -21,6 +22,10 @@ export class Feature<T extends JSONValue> {
 
 const feature = {
   feedVersion: new Feature('feed_version', 15),
+  onboardingAnimation: new Feature(
+    'onboarding_animation',
+    OnboardingAnimation.Control,
+  ),
```
Comment (nensidosari) [inline packages/shared/src/lib/featureManagement.ts:28]: I think we missed this comment of mine 😬 https://github.com/dailydotdev/apps/pull/2736#pullrequestreview-1918945730. Would u mind changing the naming and values, else I can delete and create a new one in growthbook as u've created this.

File: packages/webapp/pages/onboarding.tsx (hunk 3 above)
Comment (nensidosari) [reply packages/webapp/pages/onboarding.tsx:397]: Resolved now

Comment (sshanzel) [reply packages/webapp/pages/onboarding.tsx:397]: The old one? It should be part of the new component already.

File: packages/shared/src/lib/featureManagement.ts (hunk 4 above)
Comment (nensidosari) [reply packages/shared/src/lib/featureManagement.ts:28]: I believe versions are mostly valuable only on a/b testings, when we use them usually as follow up experiments doesn't make much sense as we set the winning version as default (not using the feature at all) and then the other as the next variation - which can be fine just as a true boolean value.
Plus this is a micro experiment so I don't think we have anything we need to relate with versions

Comment (nensidosari) [reply packages/shared/src/lib/featureManagement.ts:28]: Checking with product just in case, anyways we should be ok either way.

Comment (sshanzel) [reply packages/shared/src/lib/featureManagement.ts:28]: I mean this is an a/b testing, animations can easily be made, which means there can be follow-up experiments to this if they suddenly think of something new. But sure, let's go for boolean.

Comment (nensidosari) [reply packages/shared/src/lib/featureManagement.ts]: I don't know, don't think we use versions for most of them in their full potential. But yeah this doubt I think is also coming from not having any real guidance on how we do these. 🙌 Thanks for resolving

**Review by nensidosari — APPROVED** [review]

File: packages/shared/src/lib/featureManagement.ts (hunk 4 above)
Comment (sshanzel) [reply packages/shared/src/lib/featureManagement.ts]: Agree with that, we can maybe write a guideline in the future on how we define variables based on certain assessments.

**Pull request merged**